specs/core/1_shard-data-chains.md

```diff
@@ -157,57 +161,73 @@ def get_switchover_epoch(state: BeaconState, epoch: Epoch, index: ValidatorIndex
             % PERSISTENT_COMMITTEE_PERIOD)
 ```
 
-### `get_persistent_committee`
+### `get_shard_epoch_committee`
 
 ```python
-def get_persistent_committee(state: BeaconState,
-                             shard: Shard,
-                             slot: Slot) -> Sequence[ValidatorIndex]:
+def get_shard_epoch_committee(state: BeaconState,
```

Comment:
I think it makes sense to have two functions `get_persistent_committee` as well as `get_shard_epoch_committee`. The latter calls the former. keeping `get_persistent_committee` exposed will likely be of use to light clients.